### ISSUE TITLE
After changing book order, books don't link anymore

### ISSUE THREAD

**Issue by natyconnor** [basketbaseb/bible-fitbit#102]
When reordering the books on the book page (timeline/update for now), you can't click on them anymore. Need to figure out how they worked as links (I thought it was javascript or something). This is likely because I am recreating the divs (with their partial) now. If we can figure out how to just move the divs themselves, this will probably be fixed.

**Comment by basketbaseb** [comment]
is this still a bug?

**Comment by natyconnor** [comment]
Yes, on the Books page, you'll see a select box for ordering the books. If you change the ordering, the books will reorder but then you can't click on them. This is because I somewhat naively just create the html for the new ordering and them replace the div with it via jQuery. Never got to fixing it.

Not really a necessary feature. I was just trying to use the Freewall to do something cool but never got it to work. I wanted to reorder the divs so Freewall would animate them, but I gave up and just replaced the whole content of the page.
